### PR TITLE
VACMS-4081: Remove deprecated nickname field instances.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
`field_nickname_for_this_facility` has been deprecated. This pr removes all instances of it from vets-website.

## Testing done
Build

## Screenshots
N/A

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/stages/build/drupal/graphql/facilities-fragments/healthCareLocalFacility.node.graphql.js [hunk 1]
```diff
@@ -73,8 +72,7 @@ function queryFilter(isMainLocation) {
   return `
     filter: {conditions: [{field: "status", value: "1", operator: EQUAL, enabled: $onlyPublishedContent}, {field: "field_main_location", value: "${
       isMainLocation ? '1' : '0'
-    }", operator: EQUAL}]}, sort: {field: "field_nickname_for_this_facility", direction: ASC}
-  `;
+    }", operator: EQUAL}]}`;
```
Comment (mpelzsherman) [inline src/site/stages/build/drupal/graphql/facilities-fragments/healthCareLocalFacility.node.graphql.js:75]: Should we sort by `title` instead?

File: src/site/stages/build/process-cms-exports/transformers/node-health_care_region_page.js [hunk 2]
```diff
@@ -184,11 +182,6 @@ const transform = (
             reverseField =>
               reverseField.fieldMainLocation && reverseField.entityPublished,
           )
-          .sort((a, b) =>
-            a.fieldNicknameForThisFacility.localeCompare(
-              b.fieldNicknameForThisFacility,
-            ),
-          )
```
Comment (mpelzsherman) [inline src/site/stages/build/process-cms-exports/transformers/node-health_care_region_page.js:191]: Should we sort by title instead?

File: src/site/stages/build/process-cms-exports/transformers/node-health_care_region_page.js [hunk 3]
```diff
@@ -217,11 +209,6 @@ const transform = (
             reverseField =>
               !reverseField.fieldMainLocation && reverseField.entityPublished,
           )
-          .sort((a, b) =>
-            a.fieldNicknameForThisFacility.localeCompare(
-              b.fieldNicknameForThisFacility,
-            ),
-          )
```
Comment (mpelzsherman) [inline src/site/stages/build/process-cms-exports/transformers/node-health_care_region_page.js:224]: Sort by title?

File: src/site/stages/build/drupal/graphql/facilities-fragments/healthCareLocalFacility.node.graphql.js (hunk 1 above)
Comment (ethanteague) [reply src/site/stages/build/drupal/graphql/facilities-fragments/healthCareLocalFacility.node.graphql.js:75]: I don't think it matters - the sort logic from graphql isn't being applied anywhere, and we usually handle sorting in templates, @mpelzsherman

File: src/site/stages/build/process-cms-exports/transformers/node-health_care_region_page.js (hunk 2 above)
Comment (ethanteague) [reply src/site/stages/build/process-cms-exports/transformers/node-health_care_region_page.js:191]: I don't think it matters - the sort logic from graphql isn't being applied anywhere, and we usually handle sorting in templates, @mpelzsherman

File: src/site/stages/build/process-cms-exports/transformers/node-health_care_region_page.js (hunk 3 above)
Comment (ethanteague) [reply src/site/stages/build/process-cms-exports/transformers/node-health_care_region_page.js:224]: I don't think it matters - the sort logic from graphql isn't being applied anywhere, and we usually handle sorting in templates, @mpelzsherman

**Review by mpelzsherman — APPROVED** [review]

**Review by jhonny218 — APPROVED** [review]
LGTM

**Review by kelsonic — APPROVED** [review]
I don't think I see any issues with this PR + the deletion of `fieldNicknameForThisFacility`, but please get 👀 from @ncksllvn before merging 🎉

**Comment by kelsonic** [comment]
or not lol

**Comment by ethanteague** [comment]
> or not lol

Oh, buddy! Apologies - mashed the button before I saw your comment.

**Comment by kelsonic** [comment]
haha no problem! I've done that too 😅

**Pull request merged**